src/matched-data.ts

```diff
@@ -49,8 +49,7 @@ export function matchedData(
     .filter(validityFilter)
     .map(field => field.instance)
     .filter(locationFilter)
-    .reduce((state, instance) => _.set(state, instance.path, instance.value), {})
-    .valueOf();
```

Comment:
Had to make a fix here; missed how [`_(...).reduce()` automatically unwraps the result value](https://lodash.com/docs/4.17.15#lodash), meaning that this `valueOf()` was incorrect here.